app/test/integration_test.go

```diff
@@ -262,7 +263,14 @@ func (s *IntegrationTestSuite) TestSubmitPayForBlob() {
 			require.NoError(t, s.cctx.WaitForBlocks(3))
 
 			signer := blobtypes.NewKeyringSigner(s.cctx.Keyring, s.accounts[141], s.cctx.ChainID)
-			res, err := blob.SubmitPayForBlob(context.TODO(), signer, s.cctx.GRPCClient, []*blobtypes.Blob{tc.blob, tc.blob}, tc.opts...)
+			res, err := blob.SubmitPayForBlob(
+				context.TODO(),
+				signer,
+				s.cctx.GRPCClient,
+				sdktx.BroadcastMode_BROADCAST_MODE_BLOCK,
```

Comment:
Shouldn't this be broadcast mode sync? As per the issue?

Reply:
it was always block before, so I left it as is for now